README.md

```diff
@@ -230,8 +255,23 @@ From the repository settings page, make sure that your workflow has **Read and w
 
 <img width="152" alt="Screen Shot 2021-12-03 at 3 02 10 PM" src="https://user-images.githubusercontent.com/8978670/144665673-431e28f9-9e9d-43b3-87f8-1e5d98bed92c.png" />
 
-## Troubleshooting
+### Step 4.3: Test your production block in the Blocks app
+
+Once the action has completed, visit [https://blocks.githubnext.com/] (with no
+`devServer` parameter in the URL), then paste your repo URL
+(`https://github.com/{owner}/{repo}`) into the block picker search bar. This
+will load the production version of your block, without using your development
+server.
+
+Everything should work the same as before!
+
+### Step 4.4: Add the topic `github-blocks` to your repo (optional)
 
-- When developing folder blocks you might hit a "Something went wrong" message if you reach the rate limit of the GitHub API. This is because we're hitting the API unauthenticated. You can avoid this by configuring a personal access token (see above).
+If you want your blocks to show up in the block picker in GitHub Blocks, you need to tag this repository with the topic `github-blocks` so the application can find it.
+
+<img width="323" alt="Screen Shot 2021-12-03 at 2 54 55 PM" src="https://user-images.githubusercontent.com/8978670/144665902-63543c98-2486-4e13-9c54-f1d4bc6544a4.png" />
```

Comment:
probably worth adding a note that only people with access to the repo with the Block can see & use it, and you can keep it in a private repo.

Reply:
💯